### DISCUSSION TITLE
PCL 音乐播放还带子文件夹支持？

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#4090] [提问] [answered]
![786879f3ae133aadeaa86a548be3b8e4](https://github.com/Hex-Dragon/PCL2/assets/160083863/4e4ce8b8-ce26-4744-af19-a3888a25d291)

连临时建的子文件夹的 dll 文件都扫出来了

**Comment by WForst-Breeze** [comment, marked as answer]
是的
<!--
https://github.com/Hex-Dragon/PCL2/blob/9b6e92b9eb19563b1c3304a3eb2ec39286c003a4/Plain%20Craft%20Launcher%202/Modules/ModMusic.vb#L21-L31
